Swift/ChartboostExample.xcodeproj/project.pbxproj

```diff
@@ -379,7 +379,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 13.5;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
```

Comment:
This is a good catch. Can you set it to iOS 10.0 though? iOS 9 is being dropped with the latest SDK release

Reply:
Yes!, was about to ask that, i'll update both apps to iOS 10 🙌

Reply:
also in the pods files